data/classrooms.json

```
@@ -31,4 +31,16 @@
     "classroomName": "mango",
     "password": 777
   }
+    {
+    "classroomName": "test-m",
+    "password": 888
+  },
+    {
+    "classroomName": "test-l",
+    "password": 999
```

I like the idea of adding extra test classrooms. That'll let you demo Frempco to teachers on production without worrying that the classroom is used by anyone else.    When demoing to teachers, they might not know what a "-" is. I know its called a hyphen, but its harder to tell someone what that is.  So instead lets call them "test1", "test2" etc. 
```suggestion
    "classroomName": "test2",
    "password": 222
    "_owner_": "Leah N."
```